ramalama/model.py

```diff
@@ -622,12 +622,13 @@ def validate_args(self, args):
 
     def vllm_serve(self, args):
         exec_args = [
+            "/opt/venv/bin/python3",
```

Comment:
Is this working for the `quay.io/ramalama/*` images?

Reply:
hm I haven't tried with all ramalama images. I was assuming this is used only for vllm images which should have it https://github.com/containers/ramalama/blob/main/container-images/scripts/build-vllm.sh#L114

Reply:
Ah, its the `ramalama-vllm`. When using `--runtime vllm` it still uses the default image for my machine `quay.io/ramalama/rocm` - which doesn't include vllm. I can confirm that this work for that image. 
Would be great to (loosely) link runtime + image somehow. But not in this PR.